doc/api/http.markdown

```diff
@@ -568,6 +568,17 @@ any existing connections.
 Set to 0 to disable any kind of automatic timeout behavior on incoming
```

Comment:
Btw, speaking of which. Will setting `0` in `server.timeout` do this now? I guess it should if we would like to squeeze this commit into semver-minor. What do you think?